EventListener/RateLimitAnnotationListener.php

```diff
@@ -65,15 +63,7 @@ public function onKernelController(FilterControllerEvent $event)
             return;
         }
 
-        // Find the best match
```

Comment:
this logic has been moved in the `AnnotationLimitProcessor` as it is the one responsible for annotations